packages/remix/src/utils/serverAdapters/express.ts

```diff
@@ -52,6 +51,8 @@ function wrapExpressRequestHandler(
     }
 
     const url = new URL(request.url);
+    const pkg = await import(`${cwd()}/node_modules/react-router-dom`);
```

Comment:
can we not do this package load on every request - or cache it somehow? How about putting it in line 30, where we did it previously?